openfisca_uk/calibration/reweighting.py

```diff
@@ -0,0 +1,358 @@
+from pathlib import Path
+import h5py
+from tqdm import tqdm
+from openfisca_uk import Microsimulation
+import pandas as pd
+import numpy as np
+import tensorflow as tf
+import logging
+
+tf.get_logger().setLevel("INFO")
+logging.basicConfig(level=logging.INFO)
+
+sim = Microsimulation(adjust_weights=False)
+parameters = sim.simulation.tax_benefit_system.parameters
+statistics = parameters.calibration
+
+START_YEAR = 2019
+END_YEAR = 2022
+
+FORCE_ALL_METRIC_IMPROVEMENT = False  # Don't save weights unless all (program, year, aggregate/caseload) tests improve. If false, regressions will be logged.
+
+household_population = sim.calc("people", map_to="household").values
+household_region = sim.calc("region").values
+regions = list(pd.Series(household_region).unique())
+household_country = sim.calc("country").values
+countries = list(pd.Series(household_country).unique())
+survey_num_households = len(sim.calc("household_id"))
+
+AGGREGATE_ERROR_PENALTY = 1e-5
+PARTICIPATION_ERROR_PENALTY = 1e-1  # Person-deviations are 10,000 more loss-heavy than spending deviations per pound
+MODIFICATION_PENALTY = 1e-11
+AGE_DISTRIBUTION_PENALTY = 1e-1
```

Comment:
Looks like this is to shrink each age group's population penalty by 10, if that's right could you leave a comment?